libs/widget-lib/docs/README.md

```diff
@@ -75,99 +74,57 @@ An example of connecting a widget to Metamask:
-  textColor: string
-}
-```
+>All params are optional
+
+| Parameter              | Type                    | Description                                                                                                                                                  |
+|------------------------|-------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `width`                | `number`                | The width of the widget in pixels. Default: 400px                                                                                                            |
+| `height`               | `number`                | The height of the widget in pixels. Default: 600px                                                                                                           |
+| `metaData`             | `CowSwapWidgetMetaData` | Information about the application in which the widget is embedded. This information will help identify the source of orders and requests from users.         |
+| `provider`             | `EthereumProvider`      | The Ethereum provider to be used for interacting with a wallet.                                                                                              |
+| `chainId`              | `number`                | The blockchain ID on which the trade will take place.                                                                                                        |
+| `tradeType`            | `string`                | The type of trade. Can be `swap` or `limit-orders`.                                                                                                          |
+| `env`                  | `CowSwapWidgetEnv`      | The environment of the widget (`'local'` or `'prod'`).                                                                                                       |
+| `tradeAssets`          | `TradeAssets`           | An object containing information about the selling and buying assets.                                                                             |
+| `theme`                | `CowSwapTheme`          | The theme of the widget (`'dark'` for dark theme or `'light'` for light theme).                                                                   |
+| `logoUrl`              | `boolean`               | The width of the widget in pixels.                                                                                                                |
+| `hideLogo`             | `boolean`               | The height of the widget in pixels.                                                                                                               |
+| `hideNetworkSelector`  | `boolean`               | Disables an opportunity to change the network from the widget UI.                                                                                 |
```

Comment:
Since an integrator can preselect a network (by passing `chainId` parameter) they might want to disable the opportunity. For example, some of integrators supports only Gnosis chain, they wouldn't like their users to switch to Mainnet.
And yes, this option works only in standalone mode.

Reply:
<img width="445" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/7ba91dc1-b775-4b8f-82a3-da58c05457e8">

@fairlighteth looks like we never show the network selector in the widget mode. Could you please help with that?